### PR TITLE
Main Page UI

### PR DESCRIPTION
**Pull request by amaury901130**
#### Description:

* Main UI

---

#### Preview:

* 
Simulator Screen Shot - iPhone 11 - 2020-05-05 at 12 23 26

### CODE REVIEW

File: RSDemoProject/Home/Views/MainViewController.swift [hunk 1]
```diff
@@ -11,21 +11,34 @@ import MapKit
...
 class MainViewController: UIViewController {
   
+  let mainTitleSpacing = 1.95
+  let createTargetLabelSpacing = 1.65
+  
   var viewModel: MainViewModel!
   @IBOutlet weak var mapView: MKMapView!
+  @IBOutlet weak var mainTItle: UILabel!
+  @IBOutlet weak var createTargetLabel: UILabel!
+  @IBOutlet weak var createNewTargetButton: UIView!
```
Comment (pMalvasio) [inline RSDemoProject/Home/Views/MainViewController.swift:21]: Are you using this outlet and why is it called `Button` but is a `UIView`? 😆

File: RSDemoProject/Home/Views/MainViewController.swift [hunk 2]
```diff
@@ -11,21 +11,34 @@ import MapKit
 
 class MainViewController: UIViewController {
   
+  let mainTitleSpacing = 1.95
+  let createTargetLabelSpacing = 1.65
+  
   var viewModel: MainViewModel!
   @IBOutlet weak var mapView: MKMapView!
+  @IBOutlet weak var mainTItle: UILabel!
```
Comment (pMalvasio) [inline RSDemoProject/Home/Views/MainViewController.swift:19]: Always use the convention of putting the type of view as a suffix: `mainTitleLabel` in this case. Also, change `I` for `i`

File: RSDemoProject/Assets.xcassets/MenuBar/menuChat.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Page 1.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
```
Comment (glm4) [inline RSDemoProject/Assets.xcassets/MenuBar/menuChat.imageset/Contents.json:7]: remove 1x assets. They are not needed since our min target is iOS 11.

**Comment by glm4** [comment]
> It's me or the create target icon looks blurred?

Yes, it is *** up 🙈 
@amaury901130 Remove x1 assets and make sure the 2x and 3x are actually the right size and not burry.

File: RSDemoProject/Home/Views/MainViewController.swift [hunk 4]
```diff
@@ -16,9 +16,9 @@ class MainViewController: UIViewController {
   
   var viewModel: MainViewModel!
   @IBOutlet weak var mapView: MKMapView!
-  @IBOutlet weak var mainTItle: UILabel!
+  @IBOutlet weak var mainTitle: UILabel!
   @IBOutlet weak var createTargetLabel: UILabel!
-  @IBOutlet weak var createNewTargetButton: UIView!
+  @IBOutlet weak var createNewTarget: UIView!
```
Comment (pMalvasio) [inline RSDemoProject/Home/Views/MainViewController.swift]: Remember to keep the type of view as a suffix on the outlet like the one in the middle ` createTargetLabel`. So these two should be `mainTitleLabel` and `createNewTargetView`

**Pull request merged**